docs: update v1.10.0 settings (#1172)
longhorn/website: content/docs/1.10.0/references/settings.md

  - [Auto Cleanup Snapshot After On-Demand Backup Completed](#auto-cleanup-snapshot-after-on-demand-backup-completed)
  - [V1 Data Engine](#v1-data-engine)
  - [V2 Data Engine](#v2-data-engine)
  - [Data Engine CPU Mask](#data-engine-cpu-mask)
  - [Data Engine Hugepage Limit](#data-engine-hugepage-limit)

---

### Value Format Types by Supported Data Engines

> Each setting supports only one of the following formats, depending on its definition. The supported format determines which data engines can be configured and whether their values can differ.
> 
> - Single value for all supported Data Engines
>   - Format: Non-JSON string (e.g., `1024`)
>   - The value applies to all Data Engines supported by the setting, and the same value is used for all.
>   - Data-engine-specific values are not allowed.
> 
> - Data-engine-specific values for V1 and V2 Data Engines
>   - Format: JSON object (e.g., `{"v1":"value1","v2":"value2"}`)
>   - Allows specifying different values for V1 and V2.
> 
> - Data-engine-specific values for V1 Data Engine only
>   - Format: JSON object with `v1` key (e.g., `{"v1":"value1"}`)
>   - Only V1 Data Engine can be configured; V2 Data Engine is not affected by this setting.
> 
> - Data-engine-specific values for V2 Data Engine only
>   - Format: JSON object with `v2` key (e.g., `{"v2":"value1"}`)
>   - Only V2 Data Engine can be configured; V1 Data Engine is not affected by this setting.

Reviewer: ```suggestion
Each setting supports only one of the following formats, depending on its definition. The supported format determines which Data Engines can be configured and whether their values can differ.

- Single value for all supported Data Engines
  - Format: Non-JSON string (e.g., `1024`)
  - The value applies to all supported Data Engines and must be the same across them.
  - Data-engine-specific values are not allowed.
- Data-engine-specific values for V1 and V2 Data Engines
  - Format: JSON object (e.g., `{"v1": "value1", "v2": "value2"}`)
  - Allows specifying different values for V1 and V2 Data Engines.
- Data-engine-specific values for V1 Data Engine only
  - Format: JSON object with `v1` key only (e.g., `{"v1": "value1"}`)
  - Only the V1 Data Engine can be configured; the V2 Data Engine is not affected.
- Data-engine-specific values for V2 Data Engine only
  - Format: JSON object with `v2` key only (e.g., `{"v2": "value1"}`)
  - Only the V2 Data Engine can be configured; the V1 Data Engine is not affected.
```

Enable improved ReadWriteMany volume HA by shortening the time it takes to recover from a node failure.

#### Data Engine Log Level

> Default: `{"v2":"Notice"}`

> Applies only to the V2 Data Engine. Specifies the log level for the Storage Performance Development Kit (SPDK) target daemon. Supported values are: Error, Warning, Notice, Info, and Debug.

Reviewer: ```suggestion
Applies only to the V2 Data Engine. Specifies the log level for the Storage Performance Development Kit (SPDK) target daemon. Supported values: `Error`, `Warning`, `Notice`, `Info`, and `Debug`.
```

#### Guaranteed Instance Manager CPU

> Default: `{"v1":"12","v2":"12"}`

> Percentage of the total allocatable CPU resources on each node to be reserved for each instance manager pod. For example, 10 means 10% of the total CPU on a node will be allocated to each instance manager pod on this node. This will help maintain engine and replica stability during high node workload.

Reviewer: ```suggestion
Percentage of the total allocatable CPU resources on each node to reserve for each instance manager pod. For example, a value of `10` means 10% of the total CPU on a node will be allocated to each instance manager pod on that node. This helps maintain engine and replica stability during periods of high node workload.
```
